dist/angular-input-fields.js

```diff
@@ -179,8 +179,22 @@
         }
     ]);
 
+    angular.module('aif-add-suffix-filter', []).filter('aifAddSuffix', [
+        function () {
+            return _.memoize(function (input, suffix) {
+                if (angular.isUndefined(suffix)) {
+                    suffix = '';
+                }
+                return input + '' + suffix;
+            }, function (input, suffix) {
+                return input + '' + suffix;
```

Comment:
> I wanted to make sure it's always a string

The hash key will be stringified when used as an object key anyway, so it can be either a number or a string. If `+` will only end up with a finite number or a string, then you don't need `+ ''` in there